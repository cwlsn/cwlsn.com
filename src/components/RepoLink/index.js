import React from 'react'
import styled from 'styled-components'
import Icon from '../Icon'

const Wrapper = styled.footer`
	text-align: center;
	margin: 0 20px 40px;
	font-size: 18px;
	color: rgba(255,255,255,0.3);
`

const Copy = styled.p`
	margin: 0;
	display: flex;
	align-items: center;
	justify-content: center;
	margin-bottom: 20px;

	& > span {
		color: #c0ff93;
		margin-right: 6px;
	}
`

const Attr = styled.a`
	color: #c0ff93;
`

const RepoLink = () => (
	<Wrapper>
		<Copy>
			This site is available on GitHub if you'd like a quick start to a personal site.
		</Copy>
		<Copy>
			<Icon
				icon="github"
				size="21px"
			/>
			<Attr href="https://github.com/cwlsn/cwlsn.com">
				cwlsn/cwlsn.com
			</Attr>
		</Copy>
	</Wrapper>
)

export default RepoLink
