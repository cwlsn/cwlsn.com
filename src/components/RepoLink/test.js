import React from 'react'
import renderer from 'react-test-renderer'
import RepoLink from '.'

test('RepoLink', () => {
	const tree = renderer
		.create(<RepoLink />)
		.toJSON()

	expect(tree)
		.toMatchSnapshot()
})
