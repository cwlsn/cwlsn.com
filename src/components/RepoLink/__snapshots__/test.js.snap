// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoLink 1`] = `<test-file-stub />`;
